src/main/java/com/solvd/movie/persistence/field/impl/Country.java

```diff
@@ -0,0 +1,20 @@
+package com.solvd.movie.persistence.field.impl;
+
+import com.solvd.movie.model.criteria.SearchCriteria;
+import com.solvd.movie.persistence.field.Field;
+import org.springframework.data.elasticsearch.core.query.Criteria;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Country implements Field {
+
+    @Override
+    public void apply(final SearchCriteria searchCriteria,
```

Comment:
@hizmailovich your search criteria and criteria can be fields of Field object

Reply:
I will try